.gitignore

```diff
@@ -1 +1,14 @@
 *.swp
+.sconsign.dblite
+config.log
+.sconf_temp
+*.o
+*.os
+*.so
+*.cfgc
+*.pyc
+*_wrap.cc
+*Lib.py
```

Comment:
I don't think we should expect to find any of the above files in ts_xml except possibly lines 2-4 (.scon... and config.log) and not even those if we can ditch the SConstruct file.